.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -2,8 +2,8 @@
 </i>
 
 ### Contribution checklist:
-- [ ] I have reviewed the [documentation](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/202/Overall-Process-of-AME-Onboarding) for the workflow.
-- [ ] [Validation tools](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/84/Swagger-Validation-tools) were run on swagger spec(s) and have all been fixed in this PR. [How to fix?](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/ci-fix.md)
+- [ ] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
+- [ ] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and have all been fixed in this PR. [How to fix?]( https://aka.ms/ci-fix )
 
 If any further question about AME onboarding or validation tools, please view the [FAQ](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/FAQ.md).
```

Comment:
We should also put short link for FAQ which should point to wiki not document

Reply:
Updated